jenkins_jobs/modules/triggers.py

```diff
@@ -349,6 +350,8 @@ def gerrit(registry, xml_parent, data):
          * **draft-published-event** -- Trigger on draft published event.
          * **ref-updated-event** -- Trigger on ref-updated.
            Gerrit Trigger Plugin version >= 2.29.0
+         * **topic-changed-event** -- Trigger on topic-changed.
+           Gerrit Trigger Plugin version >= 2.30.5
```

Comment:
According to [GitHub commit](https://github.com/jenkinsci/gerrit-trigger-plugin/commit/32d6e971e6c6a89d4376bbd421622af766d3aaa4) the implementation was added in v2.26.0.